airflow/dags/ingestion/ask-astro-load-stackoverflow.py

```diff
@@ -37,13 +37,19 @@ def ask_astro_load_stackoverflow():
     any existing documents that have been updated will be removed and re-added.
     """
 
-    stack_overflow_docs = (
+    stack_overflow_archive_docs = (
```

Comment:
do we need archive if we are making call to stackoverflow directly using API?

Reply:
due to API rate limit, i think that would still be helpful when initaliing

Reply:
Got it. I though we wanted to add an additional way.

Reply:
I just removed `stack_overflow_archive_docs` and tested the API a few times. It seems that we're far from reaching the rate limit. also, I add an env var for us to config the cutoff date. it might not make that much sense to always run it from that day. or maybe we could make it an airflow variable